<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstaMsgBackupReaderClass</class>
 <widget class="QMainWindow" name="InstaMsgBackupReaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instagram BackUp Message Reader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Backup file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_backupfilelocation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadbackup">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Backup File</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txt_messages">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>691</width>
      <height>361</height>
     </rect>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Talked Friend List</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Messages</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lw_friends">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>231</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="InstaMsgBackupReader.qrc"/>
 </resources>
 <connections/>
</ui>
